src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -548,9 +548,19 @@ private List<String> identifyWinners(
       // * If this is a single-winner election in which it's possible for no candidate to reach the
       //   threshold (i.e. "first round determines threshold" is set), the tiebreaker will choose
       //   the only winner.
-      boolean useTiebreakerIfNeeded = config.isMultiSeatAllowOnlyOneWinnerPerRoundEnabled()
-          || config.isFirstRoundDeterminesThresholdEnabled();
-      if (useTiebreakerIfNeeded && selectedWinners.size() > 1) {
+      boolean needsTiebreakMultipleWinners = selectedWinners.size() > 1
+          && (config.isMultiSeatAllowOnlyOneWinnerPerRoundEnabled()
+          || config.isFirstRoundDeterminesThresholdEnabled());
+      // Edge case: there are two candidates remaining. To avoid having just one candidate in the
+      // final round, we break the tie here. Happens when we have unfilled seats, two candidates
+      // remaining, neither meets the threshold, and both have more than the minimum vote threshold.
+      boolean needsTiebreakNoWinners = config.getNumberOfWinners() == 1
+          && selectedWinners.size() == 0
+          && currentRoundTally.numActiveCandidates() == 2
+          && numSeatsUnfilled == 1
+          && currentRoundTallyToCandidates.keySet().stream().allMatch(
+              x -> x.compareTo(config.getMinimumVoteThreshold()) >= 0);
```

Comment:
This doesn't look logically wrong, but are these last two parts of the condition actually necessary? In other words, could either of these ever be false when the first three conditions are true?

Reply:
Yes, without this we had one failing test (`minimum_threshold_test`) -- since everyone under the threshold is still considered active until they're eliminated.

Reply:
I was confused by this until I realized that you meant to refer to the `no_one_meets_minimum` test. That makes sense. Maybe add an inline comment explaining why we need this condition?

It would also be helpful to explain why we need `numSeatsUnfilled == 1`, since it's also not obvious (at least to me) at first glance why it's necessary in addition to the other conditions.

Reply:
Added comments explaining it all! The numSeatsUnfilled is a check to see if the winner was selected in a previous round.